django/db/models/options.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, meta, app_label=None):
         self.verbose_name_plural = None
         self.db_table = ''
         self.ordering = []
+        self._ordering_orig = []
```

Comment:
I think so, because we don't need original ordering values, we need only a flag.